crates/sui-core/src/transaction_orchestrator.rs

```diff
@@ -116,6 +116,7 @@ where
             .start(),
         );
 
+        let disable_fastpath = std::env::var("DISABLE_FASTPATH").is_ok();
```

Comment:
I think it's better to put this in NodeConfig, such that node operators have one single file to see what options they have.

Reply:
Sure I will add it to node config instead